ext/lexbor/config.w32

```diff
@@ -26,5 +26,5 @@ if (PHP_LEXBOR == "yes") {
 
     AC_DEFINE("HAVE_LEXBOR", 1, "Define to 1 if the PHP extension 'lexbor' is available.");
 
-    PHP_INSTALL_HEADERS("ext/lexbor", "php_lexbor.h");
+    PHP_INSTALL_HEADERS("ext/lexbor", "php_lexbor.h /");
```

Comment:
Windows is still broken, based on the pcre one that cmb linked, perhaps it must be this?

```
PHP_INSTALL_HEADERS("ext/lexbor", "php_lexbor.h lexbor/");
```

Reply:
First, I tried to use exactly `lexbor/`, and then I force pushed the current attempt, but they both had the same failures.